dolphinscheduler-service/src/main/java/org/apache/dolphinscheduler/service/expand/CuringParamsServiceImpl.java

```diff
@@ -276,7 +279,8 @@ public Map<String, Property> paramParsingPreparation(@NonNull TaskInstance taskI
      */
     private Map<String, String> setBuiltInParamsMap(@NonNull TaskInstance taskInstance,
                                                     WorkflowInstance workflowInstance,
-                                                    String timeZone) {
+                                                    String timeZone,
```

Comment:
Using Project rather than WorkflowDefinition here.

Reply:
> Using Project rather than WorkflowDefinition here.

![image](https://github.com/user-attachments/assets/3f159386-2ff4-483f-a4e2-62959ed1cdaa)
Add in workflowInstance, so how about this

Reply:
Avoid add extra field to DO which is not exist in table.

Reply:
> Avoid add extra field to DO which is not exist in table.

![image](https://github.com/user-attachments/assets/7ad83499-281c-4e57-b12f-b33a1f963785)
Then go over there by passing the reference